CONTRIBUTING.md

```diff
@@ -194,8 +194,9 @@ Here is a list of the npm packages in this repository:
  | [vite-dev-server](./npm/vite-dev-server)     | `@cypress/vite-dev-server`    | Vite powered dev server for Component Testing.                  |
  | [vue](./npm/vue)                                       | `@cypress/vue`                     | Cypress component testing for Vue 3.               |
  | [vue2](./npm/vue2)                                       | `@cypress/vue2`                     | Cypress component testing for Vue 2.               |
- | [webpack-batteries-included-preprocessor](./npm/webpack-batteries-included-preprocessor)     | `@cypress/webpack-batteries-included-preprocessor`    | Cypress preprocessor for bundling JavaScript via webpack with dependencies included and support for various ES features, TypeScript, and CoffeeScript.  |
+ | [vite-dev-server](./npm/vite-dev-server)                  | `@cypress/vite-dev-server`          | Vite powered dev server for Component Testing                                                                                                          |
```

Comment:
Isn't this mention above in the same document? Line 194.
![Screenshot 2024-07-10 at 9 53 40 AM](https://github.com/cypress-io/cypress/assets/1271364/a7e04e7c-8966-4fd0-82b7-3d3fc3ea64a1)

Reply:
yes, I am so sorry for my fault.